src/EnergyPlus/Coils/CoilCoolingDXCurveFitPerformance.cc

```diff
@@ -359,9 +360,10 @@ void CoilCoolingDXCurveFitPerformance::calculate(CoilCoolingDXCurveFitOperatingM
 
 }
 
-void CoilCoolingDXCurveFitPerformance::calcStandardRatings(int supplyFanIndex, int const supplyFanType, std::string const &supplyFanName, int condInletNodeIndex, OutputFiles &outputFiles) {
-
+void CoilCoolingDXCurveFitPerformance::calcStandardRatings(EnergyPlusData &state, int supplyFanIndex, int const supplyFanType, std::string const &supplyFanName, int condInletNodeIndex, OutputFiles &outputFiles) {
 
+    using General::SolveRoot;
+    using TempSolveRoot::SolveRoot;
```

Comment:
This `using` declaration seems unnecessary as you are explicitly scoping the call to `TempSolveRoot` later?

Reply:
yes, this is temporary and will get cleaned up once we convert all of General and can remove TempSolveRoot